src/igvfd/audit/measurement_set.py

```diff
@@ -137,15 +137,15 @@ def audit_inconsistent_modifications(value, system):
             f'MeasurementSet {audit_link(path_to_text(value["@id"]),value["@id"])} has '
             f'samples with inconsistent modifications applied.'
         )
-        yield AuditFailure('inconsistent modifications', detail, level='NOT_COMPLIANT')
+        yield AuditFailure('inconsistent modifications', detail, level='ERROR')
 
 
 @audit_checker('MeasurementSet', frame='object')
 def audit_CRISPR_screen_lacking_modifications(value, system):
     '''
-        audit_detail: CRISPR screen and cas mediated mutagenesis measurement sets are required to have a modification specified on their samples.
+        audit_detail: CRISPR-based measurement sets are expected to have a modification specified on their samples.
         audit_category: missing modification
-        audit_levels: ERROR
+        audit_levels: NOT_COMPLIANT
```

Comment:
@ian-whaling is this one that you wanted to talk about?  I am not sure why this wouldn't be an error still

Reply:
Let's discuss this one.